src/OpenTelemetry.AutoInstrumentation/OpenTelemetry.AutoInstrumentation.csproj

```diff
@@ -9,6 +9,7 @@
     <PackageReference Include="OpenTelemetry.Api" Version="1.2.0" />
     <PackageReference Include="OpenTelemetry.Exporter.Console" Version="1.2.0" />
     <PackageReference Include="OpenTelemetry.Instrumentation.Http" Version="1.0.0-rc9.2" />
+    <PackageReference Include="OpenTelemetry.Instrumentation.Runtime" Version="0.1.0-alpha.1" />
```

Comment:
https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Runtime/0.1.0-alpha.1 is referencing OpenTelemetry.API 1.2.0-rc2. It is worth to ask/create PR to release alpha.2 with a newer references.

Reply:
Had discussion with Sean who is contributing to this project. There are no plans to release the next version immediately. As the dependency is [OpenTelemetry.Api](https://www.nuget.org/packages/OpenTelemetry.Api/) (>= 1.2.0-rc2) or greater. Our project will not have impact as we have explicit package reference to `OpenTelemetry.Api` 1.2.0, so this will be auto-upgraded. 

Reference: `OpenTelemetry.Api` version in `tracer-home`

![image](https://user-images.githubusercontent.com/9479006/169133304-9ba7655e-bef5-4ba3-8668-f9403458d356.png)